vars/deployPlannerSnapshot.groovy

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+
+import java.util.regex.Pattern
+
+def call(body) {
+    // evaluate the body block, and collect configuration into the object
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def yaml
+    def openShiftProject = config.openShiftProject
```

Comment:
1. I have used several times later in the code, instead of reading every time from the map I thought local var is good. Not sure though.
2. `.delegate` does kind of magic, it will export values passed to function and keep filling up the empty config with provided values if I get it right.